app/stores/CollectionsStore.js

```diff
@@ -88,6 +89,22 @@ export default class CollectionsStore extends BaseStore<Collection> {
     });
   }
 
+  search = (query: string): CollectionSearchResult[] => {
```

Comment:
Just return an array of `Collection[]` from this method, no need for the `SearchResult` object in this case as there is no ranking or context etc.